### ISSUE TITLE
Site missing default Lang

### ISSUE THREAD

**Issue by greenervigil** [greenervigil/next-portfolio#52]
Lang attribute should be added to the HTML document inorder for the screen reader to know how to pronounce certain words as they may way from eng-us to eng-uk
